src/selection.ts

```diff
@@ -247,13 +278,15 @@ export const defaultConfig: SelectionConfig = {
   multi: {
     on: 'click',
     fields: [SELECTION_ID],
+    clear: 'dblclick',
```

Comment:
Not sure why `multi` and `interval` selections have a `clear` set by default but `single` doesn't? Recall that in #4306, the second example motivated the need for clearing selections using a `single` selection. Let's have `clear: 'dblclick'` on all selection types.

Reply:
I had a quick question related to this that I wanted to follow up about. What should the `signals` array contain when I call it from the transform? I had thought it would contain all `signals` in the given `selCmpt`, but when I try to iterate through it appears some are missing. I haven't been able to figure out why this is happening.

Are transforms compiled further upstream before other parts?